Add new controller with fullscreen always option

kjthorpe18 opened the pull request:
Updates the controller with [argparse](https://docs.python.org/3/library/argparse.html) to replace the arguments scheme previously used. 

Two arguments exist for this controller, both defaulting to False (will be true if included or any value is specified for them): 
- `-e` or `--emulated`, which replaces the e argument used before and tells the app to run in an emulated mode
- `-f` or `--fullscreen`, which tells the app to run in full screen always
    - controls a conditional in `impl/apps_v2/spotify_player.py` that sets the album art to full screen, and does not modify artist, title, etc.

Commit:
Add option for always full screen

Adds argparse to replace the positional arguments scheme previously used. Two arguments exist for this controller, both defaulting to False:
-e or --emulated, which replaces the e argument used before
-f or --fullscreen, which tells the app to run in full screen always

Commit:
Add default value for fullscreen argument in spotify_player

kylejohnsonkj commented:
Looks good! I can merge it as is, but I don't see a reason to keep both controllers. Could you make a commit to overwrite the v3 controller with your v4 changes, and delete v4? Guess you wouldn't need the default value either. The original matrix-dashboard project uses that filename, so I'd like to stick with it for easy comparing.

The readme would just need minimal changes as well for how to supply the new arguments.

I could make the changes but I don't want to steal your credit. :) Thanks!

Commit:
Use only one controller, and update README

Moves changes for arguments into the v3 controller and delete the v4 controller.
Also, update the README with new instructions for running the project

kylejohnsonkj commented:
thanks for contributing 😊
